micronaut-langchain4j-core/src/main/java/io/micronaut/langchain4j/store/memory/chat/MessageWindowChatMemoryConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2025 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.langchain4j.store.memory.chat;
+
+/**
+ * Configuration for {@link dev.langchain4j.memory.chat.MessageWindowChatMemory}.
+ */
+public interface MessageWindowChatMemoryConfiguration {
+    /**
+     * @return The maximum number of messages to retain. If there isn't enough space for a new message, the oldest one is evicted.
+     */
+    Integer getMaxMessages();
```

Comment:
add a nullability annotation. Does this have a default value? In which case this can be a `default` method that returns the default value.

Reply:
[I changed the method to be a `default` method](https://github.com/micronaut-projects/micronaut-langchain4j/pull/173/commits/f501ae0484858a67e4be9baf6f0b70fb66141348)